lib/teleterm/clusters/cluster_apps.go

```diff
@@ -181,27 +182,18 @@ func (c *Cluster) reissueAppCert(ctx context.Context, clusterClient *client.Clus
 		return tls.Certificate{}, trace.Wrap(err)
 	}
 
-	err = clusterClient.ReissueUserCerts(ctx, client.CertCacheKeep, client.ReissueParams{
+	key, _, err := clusterClient.IssueUserCertsWithMFA(ctx, client.ReissueParams{
```

Comment:
As I mentioned during the call, when per-session MFA is needed, certs for local proxies are stored in memory instead of on disk. Check [RFD 90](https://github.com/gravitational/teleport/blob/master/rfd/0090-db-mfa-sessions.md) for more details.